helpers/db.js

```diff
@@ -0,0 +1,33 @@
+import mongoose, { model, models, Schema } from "mongoose";
+
+const URI = `mongodb+srv://mooowik:${process.env.MONGODB_PASSWORD}@lost-n-found.dp2d557.mongodb.net/?retryWrites=true&w=majority`;
+
```

Comment:
I prefer storing the whole thing in my .env.local. I now know your username and cluster, and see that your data in MongoDB probably runs on the "test" collection. I would also consider this data sensitive enough that it should not be easily readable in the repository.

Customizing the collection name is also cool and goes like this:

````
mongodb+srv://mooowik:${process.env.MONGODB_PASSWORD}@lost-n-found.dp2d557.mongodb.net/my-cool-collection-name?retryWrites=true&w=majority

Reply:
Hi Dominik, 
thank you for reviewing! I have only one question. If I write in evn.local my MONGODB_PASSWORD like below, it works.
```
MONGODB_PASSWORD=........
MONGODB=mongodb+srv://mooowik:$MONGODB_PASSWORD@lost-n-found.dp2d557.mongodb.net/?retryWrites=true&w=majority
```
But if I do `export MONGODB_PASSWORD=........` in terminal and remove it from evn.local like this, it don't :(
```
MONGODB=mongodb+srv://mooowik:${process.env.MONGODB_PASSWORD}@lost-n-found.dp2d557.mongodb.net/?retryWrites=true&w=majority
```
Can you give my a advice, how should I write it?

Reply:
Dont split the uri at all, just put it like this into your .env.local

```
MONGODB_URI=mongodb+srv://<username>:<password>@<cluster>.abcdefg.mongodb.net/<database>?retryWrites=true&w=majority

or using your fake pw ;) 

MONGODB=mongodb+srv://mooowik:........@lost-n-found.dp2d557.mongodb.net/?retryWrites=true&w=majority
´´´

Reply:
It don't works with Vercel preview.... I had to restore it. May be I can fix it later.